invenio_search/ext.py

```diff
@@ -405,6 +405,20 @@ def _build(tree_or_filename, alias=None):
                     name=action["alias"],
                     ignore=ignore,
                 )
+    def update_mapping(self, index):
+        """Update mapping of the existing index."""
+        mapping_path = self.mappings[index]
+        final_index = build_alias_name(index)
+
+        index_ = self.client.indices.get(final_index)
+        full_index_name = list(index_.keys())[0]
+        index_ = dsl.Index(full_index_name, using=self.client)
```

Comment:
I guess, getting here the full index name (i.e. + timestamp suffix) is necessary, so that the `put_mapping` doesn't get called with an alias?

Reply:
yes, otherwise it creates a new index